examples/starwars.ts

```diff
@@ -147,36 +149,40 @@ const episodeEnum = t.enumType({
   ],
 });
 
-const characterInterface: Interface<Context, ICharacter | null> = t.interfaceType<ICharacter>({
-  name: 'Character',
-  interfaces: [],
-  fields: () => [
-    t.abstractField('id', t.NonNull(t.ID)),
-    t.abstractField('name', t.NonNull(t.String)),
-    t.abstractField('appearsIn', t.NonNull(t.List(t.NonNull(episodeEnum)))),
-    t.abstractField('friends', characterConnectionType),
-  ],
-});
-
-const { connectionType: characterConnectionType } = relay.connectionDefinitions<ICharacter>({
-  nodeType: characterInterface,
-  edgeFields: () => [
-    t.field('friendshipTime', {
-      type: t.String,
-      resolve: (_edge: Edge<ICharacter>) => {
-        return 'Yesterday';
-      },
-    }),
-  ],
-  connectionFields: () => [
-    t.field('totalCount', {
-      type: t.Int,
-      resolve: () => {
-        return Object.keys(humanData).length + Object.keys(droidData).length;
-      },
-    }),
-  ],
-});
+const characterInterface: Interface<Context, ICharacter | null> =
+  t.interfaceType<ICharacter>({
+    name: 'Character',
+    interfaces: [],
+    fields: () => [
+      t.abstractField('id', t.NonNull(t.ID)),
+      t.abstractField('name', t.NonNull(t.String)),
+      t.abstractField('appearsIn', t.NonNull(t.List(t.NonNull(episodeEnum)))),
+      t.abstractField('friends', characterConnectionType),
```

Comment:
These abstract fields are still using the old style?

Reply:
I forgot to update these in the example, fixed it in cebe95c